src/examples/create_release.yml

```diff
@@ -7,7 +7,7 @@ usage:
   version: "2.1"
 
   orbs:
-    pipeline-feedback: instana/pipeline-feedback@1.0.0
+    pipeline-feedback: instana/pipeline-feedback@1.1
```

Comment:
Should be `1.1.1` :-)

Reply:
That was intentional. It will use the latest `1.1` version in case there is some patch applied.